### ISSUE TITLE
SPOP return multiple elements

### ISSUE THREAD

**Issue by Chandler243** [luin/ioredis#214]
It's pretty likely I'm just misreading something, but is there a way to return multiple elements with the .spop function? In native redis, I can just perform "SPOP myset 3", but with ioredis, it seems I can only return 1 element at a time (At least using the default spop function.)

**Comment by luin** [comment]
Doesn't `redis.spop("myset", 3)` work?

**Comment by Chandler243** [comment]
I tried to perform 
```
redis.spop('myset', 3, function(err, elements) {
     console.log(elements)
})
```
And it complained that I had too many arguments for the 'spop' command.

**Comment by luin** [comment]
You're running a redis server that doesn't support `count` argument. Here's a notice from the redis doc:

> The count argument will be available in a later version and is not available in 2.6, 2.8, 3.0

http://redis.io/commands/spop

**Comment by Chandler243** [comment]
Ah, looks like I need to learn to RTFM better, Thank you very much!
